index.bs

```diff
@@ -7124,12 +7213,16 @@ reason.
      null otherwise. If this throws an exception |e|, return [=a promise rejected with=] |e|.
   1. If |result| is a {{Promise}}, then return |result|.
   1. Return [=a promise resolved with=] undefined.
+ 1. Let |cancelAlgorithmWrapper| be an algorithm that runs these steps given a value |reason|:
+  1. Let |result| be the result of running |cancelAlgorithm| given |reason|, if |cancelAlgorithm|
+     was given, or null otherwise. If this throws an exception |e|, return
+     [=a promise rejected with=] |e|.
```

Comment:
The handling of `result` is missing, like we do in `transformAlgorithmWrapper` and `flushAlgorithmWrapper`:
```suggestion
 1. Let |cancelAlgorithmWrapper| be an algorithm that runs these steps given a value |reason|:
  1. Let |result| be the result of running |cancelAlgorithm| given |reason|, if |cancelAlgorithm|
     was given, or null otherwise. If this throws an exception |e|, return
     [=a promise rejected with=] |e|.
  1. If |result| is a {{Promise}}, then return |result|.
  1. Return [=a promise resolved with=] undefined.
```